lib/logstash/inputs/lumberjack.rb

```diff
@@ -126,4 +127,69 @@ def start_buffer_broker(output_queue)
       end
     end
   end
+
+  class Decoration < LogStash::Inputs::Base
+    public
+    def decorate(event)
+      super(event)
+    end
+  end # class Decoration
+
+  class Client
```

Comment:
Maybe renaming it to `ConnectionDecorator` woud be better since we are actually decorating the Lumberjack's connection class. What do you think?